animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift

```diff
@@ -188,3 +194,19 @@ extension FeedingPointDetailsModel {
         case error(String)
     }
 }
+
+fileprivate extension FeedingPointDetailsModel {
+    private func onlyFavoriteMutatedOf(_ left: FullFeedingPoint,
+                                       and right: FullFeedingPoint) -> Bool {
+        guard left != right else {
+            return false
+        }
+        
+        if left.identifier != right.identifier ||
```

Comment:
shouldn't this check be implemented over `FullFeedingPoint` itself or as an extension

Reply:
good point, will do it this way)